mne/preprocessing/tests/test_ica.py

```diff
@@ -608,4 +608,27 @@ def test_fit_params():
     ICA(fit_params=fit_params)  # test no side effects
     assert_equal(fit_params, {})
 
+
+@requires_sklearn
+def test_bad_channels():
```

Comment:
You added functionality that allows EOG to be excluded, right? If so please add a test for it to make sure it at least runs without error